src/common/components/discover-contributors/index.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useEffect, useState } from "react";
+import { History } from "history";
+import { Global } from "../../store/global/types";
+import { Account } from "../../store/accounts/types";
+import LinearProgress from "../linear-progress";
+import UserAvatar from "../user-avatar";
+import ProfileLink from "../profile-link";
+import _c from "../../util/fix-class-names";
+import { _t } from "../../i18n";
+import { syncSvg } from "../../img/svg";
+import contributors from "../../constants/contributors.json";
+import "./_index.scss";
+
+interface Props {
+  global: Global;
+  history: History;
+  addAccount: (data: Account) => void;
+}
+
+export const DiscoverContributors = (props: Props)=> {
+
+    const [list, setList] = useState<{name: string; contributes: string[]; }[]>([])
+    const [loading, setLoading] = useState(true)
+
+    useEffect(()=> {
+        shuffleContributors(contributors)
+    },[])
+
+  const shuffleContributors = (contributors: { name: string; contributes: string[]; }[]) => {
+    setLoading(true)
+    const shuffledArray = [...contributors];
+
+    for (let i = shuffledArray.length - 1; i > 0; i--) {
```

Comment:
@Adesojisouljay There is no need to shuffle array N times, just once is sufficient.

Reply:
Okay, will fix that shortly